ironfish-cli/src/commands/service/faucet.ts

```diff
@@ -126,12 +127,15 @@ export default class Faucet extends IronfishCommand {
 
     const response = await client.getAccountBalance({ account })
 
-    if (BigInt(response.content.confirmed) < BigInt(FAUCET_AMOUNT + FAUCET_FEE)) {
+    if (
+      BigInt(response.content.confirmed) <
+      BigInt(FAUCET_AMOUNT * MAX_RECIPIENTS_PER_TRANSACTION + FAUCET_FEE)
+    ) {
       if (!this.warnedFund) {
         this.log(
-          `Faucet has insufficient funds. Needs ${FAUCET_AMOUNT + FAUCET_FEE} but has ${
-            response.content.confirmed
-          }. Waiting on more funds.`,
+          `Faucet has insufficient funds. Needs ${
+            FAUCET_AMOUNT * MAX_RECIPIENTS_PER_TRANSACTION + FAUCET_FEE
+          } but has ${response.content.confirmed}. Waiting on more funds.`,
         )
```

Comment:
It seems like we could just change this to do a best effort instead of failing if any of them cannot be sent. That way if we crank up the number, it'll still send some of them if it has partial money.